### DISCUSSION TITLE
verfication error

### DISCUSSION THREAD

**Discussion by pratyusshh** [community/community#102961] [GitHub Education]
### Hi everyone,

Hi,

I'm a student and I've been using Github for my projects and researches.

Recently, I've been receiving emails stating that I've lost my student status, and Github has also sent me an email asking me to reverify my status. However, when I tried to reverify my status on the website, I received a message saying that I do not need to reverify at this time.

Hi, Linzh! You were last verified as a student on Mar 15, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

I'm confused about what to do now. May I ask for some help to guide me on how to reverify my student status on Github? I would appreciate any help or advice on this matter.

Thank you so much for your time and support.

Best regards,

**Comment by gil-son** [comment]
So, try to contact with [Github Support](https://support.github.com/)
